### PR TITLE
FakeSQS Chef Recipe for vagrant setup

### PR DESCRIPTION
**Pull request by gilleyj**
Related to [DEVOPS-156](https://imagineeasy.atlassian.net/browse/DEVOPS-156) 

Dependency of  on https://github.com/till/easybib-cookbooks/pull/1028
Dependency of https://github.com/easybib/scholar-vagrant/pull/163

### CODE REVIEW

File: fakesqs/metadata.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name 'fakesqs'
+
+supports 'ubuntu'
+
+depends 'ruby-brightbox'
```
Comment (till) [inline fakesqs/metadata.rb]: @gilleyj My only question would be what you need this ruby for? Is the one in our vagrant boxes not enough?

**Comment by till** [comment]
@gilleyj looks 👍  generally.

As for the spec, I think a simple, `expect(chef_run).to enable_supervisor_service('fakesqs')` is fine! :) The `is_aws` stuff is tested elsewhere and I would assume it works. :)

LBNL, I am not a 100% sure if you do not need to `depend` on `easybib` to ensure the helper `is_aws` is always found.

File: fakesqs/metadata.rb (hunk 1 above)
Comment (gilleyj) [inline fakesqs/metadata.rb]: Good point.  I built the recipe while doing a full on kitchen/chef/knife tutorial and this is likely **not** a requirement in our full cookbook chain.  I will test and revamp this morning.

**Comment by gilleyj** [comment]
I had originally thought that there would be a system dependency for installing support for PHP/RUBY/PYTHON/ET.AL. libraries for AWS but quickly discovered that this was NOT the case.  So this IF became an UNLESS.  My pondering here is I have been admonished by rubocop in the past for using UNLESS as opposed to IF although in this case it didn't yell at me.  So there's that.  If you have a better way of doing this sort of reverse logic (is_not_aws) let me know?

I'll add the spec this morning, thank you for the insight I appreciate it.

**Comment by till** [comment]
@gilleyj the code is fine, just add the spec and call it a day! :) 🚀

**Comment by gilleyj** [comment]
I had a broken rubocop config locally, removed and regenerated from ops source.

**Pull request merged**